releasenotes/notes/qpy-controlflow-97608dbfee5f3e7e.yaml

```
@@ -0,0 +1,29 @@
+---
+fixes:
+  - |
+    Fixed QPY serialization of :class:`.QuantumCircuit` objects that contained
+    control flow instructions. Previously if you attempted to serialize a
+    circuit containing :class:`.IFElseOp`, :class:`.WhileLoopOp`, or
+    :class:`ForLoopOp` the serialization would fail.
+    Fixed `#7583 <https://github.com/Qiskit/qiskit-terra/issues/7583>`__
```

```suggestion
    circuit containing :class:`.IfElseOp`, :class:`.WhileLoopOp`, or
    :class:`ForLoopOp` the serialization would fail.
    Fixed `#7583 <https://github.com/Qiskit/qiskit-terra/issues/7583>`__.
```

Technically this note doesn't mention `break` or `continue`, but I think you're right to skip them - they're not important, and implied by the main loop constructs.